src/components/TodoEditForm/TodoEditForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+interface Props {
+  title: string;
+  titleFieldForm: React.RefObject<HTMLInputElement>;
+  onSubmit: (val: string) => void;
+}
+
+export const TodoEditForm: React.FC<Props> = ({
+  title,
+  titleFieldForm,
+  onSubmit,
+}) => {
+  const [newTitle, setNewTitle] = useState(title);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    onSubmit(newTitle);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        data-cy="TodoTitleField"
+        type="text"
+        className="todo__title-field"
+        placeholder="Empty todo will be deleted"
+        ref={titleFieldForm}
+        value={newTitle}
+        onChange={e => setNewTitle(e.target.value)}
```

Comment:
```suggestion
        onChange={setNewTitle}
```
and you can take event.target.value inside function

Reply:
When I change this part some problem appears: 
Types of parameters 'value' and 'event' are incompatible.
    Type 'ChangeEvent<HTMLInputElement>' is not assignable to type 'SetStateAction<string>'

Reply:
Leave it as it is now